docs/resources/aws_security_group.md.erb

```diff
@@ -12,7 +12,9 @@ SGs are a networking construct which contain ingress and egress rules for networ
 
 ## Syntax
 
-An `aws_security_group` resource block uses resource parameters to search for a Security Group and then tests that Security Group. If no SGs match, no error is raised, but the `exists` matcher returns `false` and all properties will be `nil`. If more than one SG matches (due to vague search parameters), an error is raised.
+Resource parameters: group_id, group_name, id, vpc_id 
+
+An `aws_security_group` resource block uses resource parameters to search for a Security Group and then tests that Security Group. If no SGs match, no error is raised, but the `exists` matcher returns `false` and all scalar properties will be `nil`; list properties will be empty lists. If more than one SG matches (due to vague search parameters), an error is raised.
```

Comment:
I would write Security Group (SG) so you can use the SG abbreviation with wild abandon after that.